src/components/home/About.jsx

```diff
@@ -7,24 +7,23 @@ import logo from "@/public/assets/logo.svg";
 
 const About = () => {
   return (
-    <div className="flex flex-col items-center my-[4%]">
-      <div className="flex items-center justify-center px-[10%]">
-        <div className="flex w-2/5 justify-center items-start">
-          <Image src={logo} alt="Picture of logo" className="z-1" />
+    <div className="flex flex-col items-center my-4">
+      <div className="flex flex-col sm:flex-row items-center justify-center px-10">
```

Comment:
![image](https://github.com/acm-ucr/isa-website/assets/91030482/a5d528ef-f6b6-41df-a4e6-72ad1401cfe4)

desktop should not be affected when doing mobile responsiveness. i used padding percentages to position the components when i should have used width instead. refactor to use width instead of px. check for mobile responsiveness as well